backend/back/src/types/jobIds.ts

```diff
@@ -62,5 +62,9 @@ export const JobsIdentifiers: JobsIdentifiers = {
 	"linear-on-issue-create": ({ workspace }) => uniqueWebhookId("linear-on-issue-create", workspace),
 	"linkedin-create-post": ({ workflowStepId }) => uniqueJobId("linkedin-create-post", workflowStepId),
 	"miro-create-board": ({ workflowStepId }) => uniqueJobId("miro-create-board", workflowStepId),
+	"riot-on-game-end": ({ region, summoner }) => `riot-on-game-end-${region}-${summoner}`,
+	"riot-on-game-loss": ({ region, summoner }) => `riot-on-game-loss-${region}-${summoner}`,
+	"riot-on-game-win": ({ region, summoner }) => `riot-on-game-win-${region}-${summoner}`,
+	"riot-on-level-up": ({ region, summoner }) => `riot-on-level-up-${region}-${summoner}`,
```

Comment:
I would add `lol` somewhere, as it's not a Valorant game for example, which is also Riot